### DISCUSSION TITLE
Is Forking Renovate supported the same as Renovate github app ?

### DISCUSSION THREAD

**Discussion by simonbasle** [renovatebot/renovate#13514] [Request Help] [answered]
Hi,
I'm currently looking at Renovate and have experimented with it, and I really like the tool and how configurable it is.
One thing I'd really like to limit is the write-level permissions I give to the app.
I found `Forking Renovate` app, through a direct link, but I cannot find an official statement on Renovate site about this variant of the GitHub app. Is it officially supported?

It would be great to have a security model section in the docs that explains permissions for the various hosted or self-hosted variants of renovate.

I'm aware that GitHub Apps permission granularity is not great, but it look like the `Forking Renovate` permissions would suffice in our use case (no automerging, I want the bot to behave just like any external contributor).

**Comment by HonkingGoose** [comment]
## What the GitHub App page says about `forking-renovate`

The GitHub app page for `forking-renovate` [^forking-renovate] says:

> "Forking Renovate" is the sister app to Renovate Bot. The difference is that Forking Renovate operates using forks of repositories instead of as a collaborator, so needs only read-only permissions rather than read-write. It works on public (e.g. open source) repositories only.
> 
> Please see the Renovate Bot page and documentation for further details.

If you click on the `Website` link in the right sidebar of the GitHub app page, you're taken to the Whitesource landing page for Renovate. [^landing-page]

I think this is the key part:

> "Forking Renovate" is the sister app to Renovate Bot.

Sister app implies that it's the same organization, which you can verify, because the "developer" is the `renovateapp` account, of which `@rarkins` is a member. [^people] `@rarkins` is also one of the maintainers of the official "normal" Renovate app. [^contributors]

## Renovate docs do not mention `forking`

> Please see the Renovate Bot page and documentation for further details.

I searched the Renovate Docs [^docs] site for "forking" and that comes up empty.

[^forking-renovate]: https://github.com/apps/forking-renovate
[^landing-page]: https://www.whitesourcesoftware.com/free-developer-tools/renovate/
[^docs]: https://docs.renovatebot.com/
[^people]: https://github.com/orgs/renovateapp/people
[^contributors]: https://github.com/renovatebot/renovate/graphs/contributors

**Comment by rarkins** [comment, marked as answer]
Short answer is yes

**Reply by HonkingGoose** [reply]
Should we add documentation for "Forking Renovate" to our official Renovate docs then? It's tricky as a user to find out more, as I haven't found documentation that specifically covers Forking Renovate.

Do you want an issue from me to discuss this idea in more detail?

---

See issue: #13516
